tests/python/pants_test/engine/test_fs.py

```diff
@@ -66,21 +65,21 @@ def assert_walk_snapshot(self, field, filespecs_or_globs, paths, ignore_patterns
       scheduler = self.mk_scheduler(rules=create_fs_rules(), project_tree=project_tree)
       if prepare:
         prepare(project_tree)
-      result = self.execute(scheduler, Snapshot, self.specs(filespecs_or_globs))[0]
+      result = self.execute(scheduler, Snapshot, self.path_globs(filespecs_or_globs))[0]
```

Comment:
```suggestion
      result, = self.execute(scheduler, Snapshot, self.path_globs(filespecs_or_globs))
```
I usually prefer this, but I mostly see the style you've chosen in this codebase. I'm not sure if that was an intentional decision.